JustSaying.Sample.Restaurant.KitchenConsole/ConfigurationExtensions.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using Amazon;
+using Microsoft.Extensions.Configuration;
+
+namespace JustSaying.Sample.Restaurant.KitchenConsole
+{
+    public static class ConfigurationExtensions
+    {
+        private const string AWSServiceUrlKey = "AWSServiceUrl";
+
+        private const string AWSRegionKey = "AWSRegion";
+
+        public static bool HasAWSServiceUrl(this IConfiguration configuration)
+        {
+            return !string.IsNullOrWhiteSpace(configuration[AWSServiceUrlKey]);
+        }
+
+        public static Uri GetAWSServiceUrl(this IConfiguration configuration)
```

Comment:
Nit: As this is a `Uri`, could you rename to `GetAWSServiceUri()`?

Reply:
done. I went back and forth on this because the underlying config was `...ServiceUrl`

Reply:
done